src/services/gameService.js

```diff
@@ -24,10 +24,10 @@ export const newGame = async (numberOfPlayers, avatar) => {
 
 export const getGame = async (idGame, token, register, avatar) => {
     let url = `${config.apiUrl}/${idGame}${
-        register ? '?register=1' : '?token=' + token
+        register == true ? '?register=1' : '?token=' + token
```

Comment:
It's better to cast register to a boolean instead of using `==`. I prefer to avoid `==` in general.

https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons